Simulation/6DOF_RK4/plotter/altsim.py

```diff
@@ -0,0 +1,84 @@
+# File for onbaording sim
+import matplotlib.pyplot as plt
+import numpy as np
+import time
+
+
+class Sim:
+    def __init__(self, mass, thrust, cDrag, crossA, motorMass, dm, dt, x0, temp0, pressure0):
+        self.mass = mass
+        self.thrust = thrust
+        self.cDrag = cDrag
+        self.crossA = crossA
+        self.dm = dm
+        self.dt = dt
+        self.temp0 = temp0 + 274.15
+        self.pressure0 = pressure0
+        self.motorMass = motorMass
+        self.time = 0
+        self.x = x0
+        self.v = 0
+        self.a = 0
+        self.G = 6.6743/(10**11)
+        self.mEarth = 5.97219*(10**24)
+        self.rEarth = 6.378137*(10**6)
+        self.fNet = 0
+
+    def position(self):
+        self.a = self.fNet/self.mass
+        self.v += self.a*self.dt
+        self.x += self.v*self.dt
+        
+    def aDrag(self):
+        self.temp = self.temp0 - .0065*self.x
+        if(self.temp > 0):
+            self.pressure = self.pressure0*((self.temp0/self.temp)**((9.81*.029)/(8.314*.0065)))
```

Comment:
typically you want to replace constants with variable names, so we can keep track of what they mean